docs/reference/data-sources/push.md

```diff
@@ -0,0 +1,39 @@
+# Push source
+
+## Description
+
+Push sources allow feature values to be pushed to the online store in real time. This allows fresh feature values to be made available to applications. Push sources supercede the 
+[FeatureStore.write_to_online_store](https://rtd.feast.dev/en/latest/index.html#feast.feature_store.FeatureStore.write_to_online_store).
+
+Push sources can be used by multiple feature views. When data is pushed to a push source, feast propagates the feature values to all the consuming feature views.
+
+## Example
+### Defining a push source
+
+```python
+from feast import PushSource, ValueType, BigQuerySource, FeatureView, Feature
+
+push_source = PushSource(
+    name="push_source",
+    schema={"user_id": ValueType.INT64, "life_time_value": ValueType.INT64},
+    batch_source=BigQuerySource(table="test.test"),
```

Comment:
maybe add some notes on why there is a `batch_source` here?

Reply:
I've added this to the doc string but i'll add some deets here too